src/components/paging/Paging.js

```diff
@@ -0,0 +1,62 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+export const withPaging = ComponentToPage => {
+    class WithPaging extends Component {
+      state ={
```

Comment:
```suggestion
      state = {
```
Remember to try and keep syntax consistent across files.